### ISSUE TITLE
Add comment to top of comment list immediately after submission

### ISSUE THREAD

**Issue by johnsBeharry** [peakshift/makers.bolt.fun#133]
Current Behaviour
- When a comment is posted
- Then it does not show up in the comment list

Expected Behaviour
- When comment is posted
- And it is being sent to relays
- Then it shows up immediately on the comments list

**Comment by bitcoinuser** [comment]
Agree. I posted a comment and I needed to refresh the page to see the comment.
